libparsec/crates/crypto/src/rustcrypto/sequester.rs

```diff
@@ -317,11 +317,13 @@ impl SequesterVerifyKeyDer {
         let (signature, data) =
             deserialize_with_armor(data, self.size_in_bytes(), Self::ALGORITHM)?;
 
-        // TODO: It Seems to be a mistake from RustCrypto/RSA
-        // Why should we allocate there ?
-        self.0
-            .verify(data, &Signature::from(signature.to_vec()))
-            .map_err(|_| CryptoError::SignatureVerification)?;
+        Signature::try_from(signature)
+            .map_err(|_| CryptoError::Signature)
```

Comment:
This is  a new kind  of error that can occur, do we have a corresponding test ? how is it handled in the sodiumoxide implementation ?

Reply:
`Signature::try_from` won't raise an error from the [code implementation](https://docs.rs/rsa/0.8.2/src/rsa/pss.rs.html#135-143) so I wouldn't be able to provide a test that fail to parse a raw signature bytes

Reply:
https://github.com/RustCrypto/RSA/issues/349

Reply:
a comment should be added to explain this then